### ISSUE TITLE
Custom Select Field only displays first 3 options

### ISSUE THREAD

**Issue by mrlinnth** [crater-invoice/crater#676]
**Describe the bug**

Added a "Select Field" type custom field to invoice with 5,6 options. Only 3 options show up at invoice form. The remaining options can be selected by using arrow key. 

**Expected behavior**

UI should display all added options for the user to select.

**Screenshots**

![image](https://user-images.githubusercontent.com/4952597/147276423-9ee02b12-8c09-47a7-9fc8-5778b311c318.png)

![image](https://user-images.githubusercontent.com/4952597/147276451-0b871646-eccd-425d-b30d-f41d292bf3c7.png)

**Please complete the following information:**
- Crater version: 5.0.6
- PHP version: 7.4.3
- Database type and version: MySQL 8.0.27-0ubuntu0.20.04.1 for Linux on x86_64

**Optional info**
- OS:  Linux Mint 20.2
- Browser: Microsoft Edge

**Comment by klementj** [comment]
Hi marvelefe.

Thank you for finding a bug and coming up with a solution ( PR #678 ) . While doing a review I started debugging the problem. It looks like the problem is not with the dropdown, but with the invoice page having a fixed height of 100vh and then the section with items, notes and custom fields having overflow:hidden. This cuts of of the rest of the ul with the options off, and never showing a scrollbar. 

![Here is the overflow hidden](https://user-images.githubusercontent.com/222317/152867692-7e952408-27cc-440d-a867-5e4b0d36846d.png)

Even if the page height is set to min-height: 100vh a scrollbar would appear on the page when the dropdown is opened. This is an old problem with custom dropdowns. They need to detect when they are at bottom of page and show options above. Dropdown should poperly be rewritten with JS to detect or use the [HTML select element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select). I'll open an issue for this later, for now I think we should just go with the quick fix that you provided. What does @mohitpanjwani think?

**Comment by mohitpanjwani** [comment]
> Hi marvelefe.
> 
> Thank you for finding a bug and coming up with a solution ( PR #678 ) . While doing a review I started debugging the problem. It looks like the problem is not with the dropdown, but with the invoice page having a fixed height of 100vh and then the section with items, notes and custom fields having overflow:hidden. This cuts of of the rest of the ul with the options off, and never showing a scrollbar.
> 
> ![Here is the overflow hidden](https://user-images.githubusercontent.com/222317/152867692-7e952408-27cc-440d-a867-5e4b0d36846d.png)
> 
> Even if the page height is set to min-height: 100vh a scrollbar would appear on the page when the dropdown is opened. This is an old problem with custom dropdowns. They need to detect when they are at bottom of page and show options above. Dropdown should poperly be rewritten with JS to detect or use the [HTML select element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select). I'll open an issue for this later, for now I think we should just go with the quick fix that you provided. What does @mohitpanjwani think?

Agreed with you. I was thinking of rewriting the select component implementation to use popperjs so it works regardless of the overflow or a simpler solution would be to use teleport to attach the dropdown directly on body. 

For now I’ll merge the given PR if it seems to do the work temporarily.
